JitsiMeetJS.ts

```diff
@@ -421,6 +429,35 @@ export default {
             });
     },
 
+    /**
+     * Manually create JitsiLocalTrack's from the provided track info, by exposing the RTC method
+     *
+     * @param {Array<ICreateLocalTrackFromMediaStreamOptions>} tracksInfo - array of track information
+     * @returns {Array<JitsiLocalTrack>} - created local tracks
+     */
+    createLocalTracksFromMediaStreams(tracksInfo) {
+        return RTC.createLocalTracks(tracksInfo.map((trackInfo) => {
+            const tracks = trackInfo.stream.getTracks()
+                .filter(track => track.kind === trackInfo.mediaType);
+
+            if (!tracks || tracks.length === 0) {
+                throw new JitsiTrackError(JitsiTrackErrors.TRACK_NO_STREAM_TRACKS_FOUND, null, null);
+            }
+
+            if (tracks.length > 1) {
+                throw new JitsiTrackError(JitsiTrackErrors.TRACK_TOO_MANY_TRACKS_IN_STREAM, null, null);
+            }
+
+            trackInfo.track = tracks[0];
+
+            if (!trackInfo.sourceId) {
+                trackInfo.sourceId = 'GENERATEDVALUEHERE';
```

Comment:
`sourceId` is the ID of the desktop sharing source and is used only for remote control feature in the Electron app. What purpose does it serve there?

Reply:
No purpose, happy to remove if not needed

Reply:
Just updated the docs and removed the sourceId on the new function